Sources/RingPublishingTracking/Public/Model/TrackingIdentifier.swift

```diff
@@ -11,9 +11,25 @@ import Foundation
 /// Tracking identifier and its expiration date
 public struct TrackingIdentifier {
 
-    /// Identifier
-    public let identifier: String
+    public struct EaUUID {
```

Comment:
could we have those models in separate files? just public stuff, codable entities can be like you did them